modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -1235,120 +1247,85 @@ protected function get_initial_config(): array {
 	protected function content_template_single_repeater_item() {
 		?>
 		<#
-		const tabCount = view.collection.length + 1,
-			elementUid = view.getIDInt().toString().substring( 0, 3 ) + tabCount,
-			tabIcon = elementor.helpers.renderIcon( view, data.tab_icon, { 'aria-hidden': true }, 'i' , 'object' );
-
-		let tabActiveIcon = tabIcon,
-			tabId = 'e-n-tab-title-' + elementUid;
+		const tabIndex = view.collection.length,
+			elementUid = view.getIDInt().toString().substr( 0, 3 ),
+			item = data,
+			hoverAnimationClass = view?.container?.settings?.attributes?.hover_animation
+				? `elementor-animation-${ view.container.settings.attributes.hover_animation }`
+				: '';
+		#>
+		<?php $this->content_template_single_item( '{{ tabIndex }}', '{{ item }}', '{{ elementUid }}', '{{ hoverAnimationClass }}' );
+	}
 
-		if ( '' !== data.tab_icon_active.value ) {
-			tabActiveIcon = elementor.helpers.renderIcon( view, data.tab_icon_active, { 'aria-hidden': true }, 'i' , 'object' );
-		}
+	protected function content_template() {
+		?>
+		<# const elementUid = view.getIDInt().toString().substr( 0, 3 ); #>
+		<div class="e-n-tabs" data-widget-number="{{ elementUid }}" aria-label="<?php echo esc_html__( 'Tabs. Open items with Enter or Space, close with Escape and navigate using the Arrow keys.', 'elementor' ); ?>">
+			<# if ( settings['tabs'] ) { #>
+			<div class="e-n-tabs-heading" role="tablist">
+				<# _.each( settings['tabs'], function( item, index ) {
+					const tabIndex = index,
+					hoverAnimationClass = settings['hover_animation']
+						? `elementor-animation-${ settings['hover_animation'] }`
+						: '';
+				#>
+				<?php $this->content_template_single_item( '{{ tabIndex }}', '{{ item }}', '{{ elementUid }}', '{{ hoverAnimationClass }}' ); ?>
+				<# } ); #>
+			</div>
+			<div class="e-n-tabs-content"></div>
+			<# } #>
+		</div>
+		<?php
+	}
 
-		const tabWrapperKey = {
-			'id': 'e-n-tab-title-' + elementUid,
-			'class': [ 'e-n-tab-title' ],
+	private function content_template_single_item( $tab_index, $item, $element_uid, $hover_animation_class ) {
+		?>
+		<#
+		const tabCount = tabIndex + 1,
+			tabId = item.element_id
+				? item.element_id
+				: 'e-n-tab-title-' + elementUid + ( tabIndex + 1 ),
+			tabUid = elementUid + tabCount,
+			tabIcon = elementor.helpers.renderIcon( view, item.tab_icon, { 'aria-hidden': true }, 'i' , 'object' ),
+			activeTabIcon = item.tab_icon_active.value
+				? elementor.helpers.renderIcon( view, item.tab_icon_active, { 'aria-hidden': true }, 'i' , 'object' )
+				: tabIcon;
+
+		view.addRenderAttribute( 'e-n-tab-title', {
```

Comment:
Now that we overwrite the attribute, we don't need a unique attribute key anymore.